src/commands/projects.ts

```diff
@@ -75,6 +79,24 @@ export const builder = (argv: yargs.Argv) => {
             describe: projectCreateRequest['project.name'].description,
             type: 'string',
           },
+          ipAllowIps: {
+            describe:
+              projectUpdateRequest['project.settings.allowed_ips.ips']
+                .description ??
+              'A list of IP addresses that are allowed to connect to the endpoint.',
+            type: 'string',
+            array: true,
+            group: 'IP Allow:',
```

Comment:
With `group` option we can show multiple parameters under single group as below - 

<img width="1687" alt="ip-allow-cli-group-options" src="https://github.com/neondatabase/neonctl/assets/1949852/b8065d0b-1f25-4840-aee0-c9a8e2319286">

Reply:
oh, I am afraid group option breaks the help formatting, so on narrow screen it is not even readable. Compare to other commands output

Reply:
We have a custom formatter for help message to conform with designs, it probably doesn't account for custom groups :(

Reply:
Yes, you are right. Good catch. Removed the grouping for now.